script.js

```diff
@@ -0,0 +1,360 @@
+    let invertedArray = [];
+    for (let i = 0; i < array.length; i++) {
+        const invertedNumber = array[i] * -1;
+        invertedArray.push(invertedNumber);
+    }
+    return invertedArray;
+}
+
+console.log(getInvertedArray([1, -2, 3, -4, 5]));
+
+// 16. Write a function that takes an array of numbers and returns the sum of the numbers.
+// The numbers can be negative or non-integer.
+// If the array does not contain any numbers then you should return 0.
+
+function getSumOfNumbers(numbers) {
+    let sumOfNumbers = 0;
+    if (numbers.length === 0) {
+        return 0;
+    }
+    for (let i = 0; i < numbers.length; i++) {
+        sumOfNumbers = sumOfNumbers + numbers[i];
+    }
+    return sumOfNumbers;
+}
+
+console.log(getSumOfNumbers([]));
+
+// 17. I'm new to coding and now I want to get the sum of two arrays... Actually the sum of all their elements. I'll appreciate for your help.
+// P.S. Each array includes only integer numbers. Output is a number too.
+
+// create the getArraySum function that sums the elements of a single array and use it in the arrayPlusArray function.
+
+function getArraySum(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+        sum += array[i];
+    }
+    return sum;
+}
+
+function getTheSumOfTwoArrays(firstArray, secondArray) {
+    const firstArraySum = getArraySum(firstArray);
+    const secondArraySum = getArraySum(secondArray);
+    return firstArraySum + secondArraySum;
+}
+
+console.log(getTheSumOfTwoArrays([1, 2, 3], [4, 5, 6]));
+
+// 18. Given a non-negative integer, 3 for example, return a string with a murmur: "1 sheep...2 sheep...3 sheep...".
+// Input will always be valid, i.e. no negative integers.
+
+function getSheepCount (number){
+    let murmur = '';
+    for (let i = 1; i <= number; i++) {
+        murmur += i + ' sheep...';
+    }
+    return murmur;
+}
+
+console.log(getSheepCount(10));
+
+// 19. It's the academic year's end, fateful moment of your school report. The averages must be calculated.
+// All the students come to you and entreat you to calculate their average for them. Easy ! You just need to write a script.
+// Return the average of the given array rounded down to its nearest integer.
+// The array will never be empty.
+
+function getAverage(marks){
+    let sumOfMarks = 0;
+    for (let i = 0; i < marks.length; i++) {
+        sumOfMarks += marks[i];
+    }
+    const averageOfMarks = sumOfMarks / marks.length;
+    return Math.floor(averageOfMarks);
+}
+
+console.log(getAverage([2, 3, 4, 5]));
+
+// 20. You take your son to the forest to see the monkeys.
+// You know that there are a certain number there (n), but your son is too young to just appreciate the full number, he has to start counting them from 1.
+// As a good parent, you will sit and count with him. Given the number (n), populate an array with all numbers up to and including that number, but excluding zero.
+
+function getMonkeyCount(number) {
+    let countedMonkey = [];
```

Comment:
Use plural `monkeys` instead of `monkey` - we are counting multiple monkeys here.